uiTestHelpers/mobileStepDefinitions/actions/clickElementInsideElement.js

```diff
@@ -1,5 +1,6 @@
-// module.exports = function clickElementInsideElement(locatorKey2, locatorKey) {
-//   return this.getCurrentPage()
-//     .getElementWhenInDOM(locatorKey, locatorKey2)
-//     .then((element) => element.click());
-// };
+module.exports = async function clickElementInsideElement(locatorKey2, locatorKey) {
+  const pageObj = await this.getCurrentPage();
+  const el = await pageObj.getElementInsideElement(locatorKey, locatorKey2);
```

Comment:
For consistency, shouldn't the clickElementInsideElement function take the arguments in the same order as getElementInsideElement? Like key1, key2

Reply:
The reason for this is that it reads better imo to say `click something inside something` in the step definition but the arg order should be up to n levels starting from the top of the tree. It's essentially reversed. In case you in future wanted `click something inside something inside something`

Reply:
see other comment